docs/executing_syzkaller_programs.md

```diff
@@ -47,12 +48,26 @@ Several useful `syz-execprog` flags:
     	use threaded mode in executor (default true)
 ```
 
-If you pass `-threaded=0 -collide=0`, programs will be executed as a simple single-threaded sequence of syscalls. `-threaded=1` forces execution of each syscall in a separate thread, so that execution can proceed over blocking syscalls. `-collide=1` forces second round of execution of syscalls when pairs of syscalls are executed concurrently.
+If you pass `-threaded=0`, programs will be executed as a simple single-threaded
+sequence of syscalls. `-threaded=1` forces execution of each syscall in a
+separate thread, so that execution can proceed over blocking syscalls.
```

Comment:
Single line would be better.

Reply:
That was intentional - to preserve the readability of the document from the source file. Compiled Markdown will treat that as a single paragraph anyway - we would have to put two new lines in a row to get a visible new line.